docs/source/algorithms/ComputeIncoherentDOS-v1.rst

```diff
@@ -0,0 +1,88 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Computes the phonon density of states from an inelastic neutron 
+scattering measurement of a powder or polycrystalline sample,
+assuming that all scattering is incoherent, using the formula 
+(Thermodynamic Properties of Solids, eds. Chaplot, Mittal, Choudhury,
+Chapter 3) for the 1-phonon incoherent scattering function:
+
+:math:`S^{(1)}_{\mathrm{inc}}(Q,E) = 
+   \exp\left(-2\bar{W}(Q)\right) \frac{Q^2}{E} 
+   \langle n+\frac{1}{2}\pm\frac{1}{2} \rangle
+   \left[ \sum_k \frac{\sigma_k^{\mathrm{scatt}}{2m_k} g_k(E) \right]`
```

Comment:
Sorry... 

:math:`S^{(1)}_{\mathrm{inc}}(Q,E) = \exp\left(-2\bar{W}(Q)\right) \frac{Q^2}{E} \langle n+\frac{1}{2}\pm\frac{1}{2} \rangle \left[ \sum_k \frac{\sigma_k^{\mathrm{scatt}}}{2m_k} g_k(E) \right]`

I should have checked this but I don't have DocUtils installed...